### PR TITLE
Adding a Getting Started section to the README

### PR DESCRIPTION
**Pull request by belgaied2**
**What this PR does / why we need it**:
This PR comes with the first release. It updates the documentation by providing a **Getting Started** section in the README.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**:

- [X] squashed commits into logical changes
- [X] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,14 +8,186 @@
...
+export CLUSTER_NAME=capd-rke2-test
+export CABPR_CP_REPLICAS=3
+export CABPR_WK_REPLICAS=2
+export KUBERNETES_VERSION=v1.24.6 
+```
+
+The next step is to substitue the values in the YAML using the following commands:
+
+```bash
+cat rke2-sample.yaml | envsubst > rke2-docker-example.yaml
```
Comment (richardcase) [inline README.md:128]:
```suggestion
cat rke2-sample.yaml | clusterctl generate yaml > rke2-docker-example.yaml
```

File: README.md [hunk 2]
```diff
@@ -8,14 +8,186 @@
...
+Installing Provider="control-plane-rke2" Version="v0.1.0-alpha.1" TargetNamespace="rke2-control-plane-system"
+
+Your management cluster has been initialized successfully!
+
+You can now create your first workload cluster by running the following:
+
+  clusterctl generate cluster [name] --kubernetes-version [version] | kubectl apply -f -
+```
+### Create a workload cluster
+There are some sample cluster templates available under the `samples` folder. For this `Getting Started` section, we will be using the `docker` samples available under `samples/docker/oneline-default` folder. This folder contains a YAML template file called `rke2-sample.yaml` which contains environment variable placeholders which can be substituted using the `envsubst` tool (generally available in your package manager. If not, you might want to use [this version](https://github.com/a8m/envsubst/releases)).
```
Comment (richardcase) [inline README.md:107]:
```suggestion
There are some sample cluster templates available under the `samples` folder. For this `Getting Started` section, we will be using the `docker` samples available under `samples/docker/oneline-default` folder. This folder contains a YAML template file called `rke2-sample.yaml` which contains environment variable placeholders which can be substituted using the [envsubst](https://github.com/a8m/envsubst/releases) tool. We will use `clusterctl` to generate the manifests from these template files.
```

**Review by richardcase — COMMENTED** [review]
Looks good, a coupld of small suggestions.

File: README.md (hunk 2 above)
Comment (belgaied2) [reply README.md:107]: Cool Thanks for the suggestion!

File: README.md (hunk 1 above)
Comment (belgaied2) [reply README.md:128]: Zhanks

**Comment by belgaied2** [comment]
Ignoring new linting issues, caused by recent linting configuration changes.

**Pull request merged**